src/components/BottomNav.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <div>Active: {{ bottomNav }} </div>
+    <div>Coach: {{ coach }} </div>
+    <div>Manager: {{ manager }} </div>
+    <div>Admin: {{ admin }} </div>
+    <v-bottom-nav
+      :active.sync="bottomNav"
+      :value="true"
+      fixed
+    >
+      <v-btn
+        v-if="coach"
+        color="teal"
+        flat
+        value="Coach Page"
+      >
+        <span v-show="$vuetify.breakpoint.mdAndUp">Coaches</span>
+        <v-icon>mdi-football</v-icon>
+      </v-btn>
+
+      <v-btn
+        v-if="manager"
+        color="teal"
+        flat
+        value="Regional Manager Page"
+      >
+        <span v-show="$vuetify.breakpoint.mdAndUp">Regional Managers</span>
+        <v-icon>mdi-sitemap</v-icon>
+      </v-btn>
+
+      <v-btn
+        v-if="admin"
+        color="teal"
+        flat
+        value="Admin Page"
+      >
+        <span v-show="$vuetify.breakpoint.mdAndUp">Administrators</span>
```

Comment:
```suggestion
        <span v-show="$vuetify.breakpoint.smAndUp">Coaches</span>
```
Should be sm and up due to tablets